Makefile

```diff
@@ -59,7 +59,7 @@ ifeq ($(EVT),1)
 else
 	# include directories (absolute or relative paths to additional folders with
 	# headers, current folder is always included)
-	INC_DIRS_CROSS = inc/ inc/fsae_drivers/ ../lpc11cx4-library/lpc_chip_11cxx_lib/inc ../lpc11cx4-library/evt_lib/inc/ ../MY17/lib/MY17_Can_Library/
+	INC_DIRS_CROSS = inc/ inc/fsae_drivers/ ../Opel_EE/lpc11cx4-library/lpc_chip_11cxx_lib/inc ../Opel_EE/lpc11cx4-library/evt_lib/inc/ ../MY17/lib/MY17_Can_Library/

```

Comment:
This is computer/directory setup specific, please remove the diff before merging.